docs/decisions/0012-system-taxonomy-creation.rst

```diff
@@ -0,0 +1,44 @@
+12. System-defined Taxonomy & Tags creation
+============================================
+
+Context
+--------
+
+The System-defined are closed taxonomies created by the system. Some of this are totally static (e.g Language)
+and some depends on a core data model (e.g. Organizations). It is necessary to define how to create and validate 
+the System-defined taxonomies and their tags.
+
+
+Decision
+---------
+
+Create a ``Content-side`` class that lives on ``openedx.features.tagging``.
+You must create a content-side model for each content and taxonomy to be used.
+You can use ``ContentSystemTaxonomyMixin`` and the mixin of the Taxonomy (e.g. ``LanguageSystemTaxonomy``) to create the new class.
+Then, all system-defined taxonomies must be created initially in a fixture.
+
+Each Taxonomy mixin has ``get_tags``; to configure the valid tags, and ``validate_tags``; to check if a list of tags are valid.
```

Comment:
> From what I understand, most system taxonomies won't need to override get_tags. Language is an exception, since we're filtering down the available ISO languages using Django settings.LANGUAGES , but from what I understand, most system taxonomies won't need to do this.

@pomegranited Yes it's correct. I will update the docs so this is not misunderstood

> I'd removed validate_tags from the Taxonomy data architecture, and instead proposed just validating individual object tags with [validate_object_tag](https://docs.google.com/document/d/13zfsGDfomSTCp_G-_ncevQHAb4Y4UW0d_6N8R2PdHlA/edit#heading=h.78ofj8tmnjio).

Ohh Ok, Noted :+1: